code/Controllers/OldPageRedirector.php

```diff
@@ -10,9 +10,11 @@
 use SilverStripe\Control\HTTPResponse_Exception;
 use SilverStripe\Core\Extension;
 
+/**
+ * @extends Extension<Controller>
```

Comment:
```suggestion
 * @extends Extension<ContentController|ModelAsController>
```

Reply:
Those are both subclasses of `Controller` - and there's no reason this couldn't be applied to someone's custom controller.

But given that would be a pretty unlikely edge case, and for the sake of avoiding ping pong, I've made the change.